### PR TITLE
deploy script

### PR DESCRIPTION
**Pull request by nerik**

### CODE REVIEW

File: bin/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+else
+  SERVER_URL='staging.trase.earth'
+fi
+
+echo "DEPLOYING:  $ENV_TYPE ($SERVER_URL)"
+
+read -p "Press enter to continue"
+
+cp .env .env.tmp
+cp .env.$ENV_TYPE .env
```
Comment (tiagojsag) [inline bin/deploy.sh]: https://www.cyberciti.biz/faq/unix-linux-test-existence-of-file-in-bash/

File: bin/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+npm run webpack:prod
+
+cp .env.tmp .env
+rm .env.tmp
+
+scp -pr ./dist/* ubuntu@$SERVER_URL:/var/www/html/
```
Comment (tiagojsag) [inline bin/deploy.sh]: you should restart nginx varnish and apache after this. see the command I left you on slack

File: package.json [hunk 3]
```diff
@@ -12,7 +12,9 @@
     "eslint": "npm run lint-js",
     "lint-js": "eslint -c .eslintrc.js --ext .jsx,.js scripts",
     "lint-sass": "sass-lint -c .sass-lint.yml -v -q 'styles/**/*.scss'",
-    "postinstall": "npm run build"
+    "postinstall": "npm run build",
+    "deploy:staging": "./bin/deploy.sh staging",
```
Comment (tiagojsag) [inline package.json:16]: can't we do a single command, that accepts produtcion/staging as an argument?

File: bin/deploy.sh (hunk 2 above)
Comment (tiagojsag) [reply bin/deploy.sh]: does this clean /dist before it starts?

Comment (nerik) [reply bin/deploy.sh]: > you should restart nginx varnish and apache after this. see the command I left you on slack

Even if we are just touching static files?

File: package.json (hunk 3 above)
Comment (nerik) [reply package.json:16]: Actually I thought wrapping this would prevent running `./bin/deploy.sh monkey`. Since we only have 2 envs...

**Pull request merged**